jetty-home/src/main/resources/modules/logging-log4j1.mod

```diff
@@ -0,0 +1,57 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Configure jetty logging to use Log4j Logging
+SLF4J is used as the core logging mechanism.
+
+[tags]
+logging
+
+[depends]
+logging-slf4j
+resources
+
+[provides]
+logging
+log4j
+
+[files]
+basehome:modules/logging/log4j1
+maven://org.slf4j/slf4j-log4j12/${slf4j.version}|lib/logging/slf4j-log4j12-${slf4j.version}.jar
+maven://log4j/log4j/${log4j.version}|lib/logging/log4j-${log4j.version}.jar
+
+[lib]
+lib/logging/slf4j-log4j12-${slf4j.version}.jar
+lib/logging/log4j-${log4j.version}.jar
+
+[ini]
+log4j.version?=1.2.17
+jetty.webapp.addSystemClasses+=,org.slf4j.,org.apache.log4j.
+
+
+[license]
+SLF4J is distributed under the MIT License.
```

Comment:
This is the wrong license.   The license here should be for the log4j jar downloaded.  The `logging-slf4j` module should have the slf4j license.